### DISCUSSION TITLE
User not authorized

### DISCUSSION THREAD

**Discussion by julioxavierr** [community/community#19091] [Copilot Conversations] [answered]
I've been using copilot for more than six months as part of the preview and I love it.

However, it stopped working with VSCode 1.68.1 today after getting the notification about the changes regarding the pricing. 

When I first open VSCode it asks me to sign up:

![Screen Shot 2022-06-21 at 5 43 16 PM](https://user-images.githubusercontent.com/14119096/174894696-0e3e8a3e-1d12-4979-bbba-d726e65c11b7.png)

And that redirects me to the billing page (https://github.com/settings/billing)

If I click on the copilot icon, on the bottom right, it gives me the following error:

![Screen Shot 2022-06-21 at 5 43 27 PM](https://user-images.githubusercontent.com/14119096/174894865-0cb8493d-d32a-4aa0-91b3-09b386c83629.png)

```
[INFO] [auth] [2022-06-21T20:43:11.483Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-21T20:43:11.483Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Comment by ldeluigi** [comment]
It has become mandatory to either pay or use the free trial.

Nevertheless, if you are a student, it should be free. Currently there are some [issues](#19050), though.

**Reply by joekadi** [reply]
So does everyone who's not a student that was using beta for months will now need to pay or use trial?

**Reply by Arnish-B** [reply]
how do we use github copilot as a student?

**Comment by julioxavierr** [comment, marked as answer]
Just in case anyone experiences something similar in the future:

My account's billing was in an odd state - there was a failure related to a billing attempt from 2020. I contacted GitHub support and it got fixed.
